_posts/0000-01-02-aidenhernandez.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Hold on to your butts!

![spiderman_pooping_face](https://user-images.githubusercontent.com/72896752/98977883-fa3b0680-24cd-11eb-8d65-d17576ff8137.gif)